docs/getting-started-guides/index.md

```diff
@@ -48,7 +48,7 @@ few commands, and have active community support.
 
 - [GCE](/docs/getting-started-guides/gce)
 - [AWS](/docs/getting-started-guides/aws)
-- [Azure](/docs/getting-started-guides/coreos/azure/)
+- [Azure](/docs/getting-started-guides/azure)
```

Comment:
Yeah, let's not remove the other one.  But we should add some information so people can distinguish them. How about something like
```
[Azure](/docs/getting-started-guides/coreos/azure/) (Weave-based, contributed by WeaveWorks employees)
[Azure](/docs/getting-started-guides/azure/) (Flannel-based, contributed by Microsoft employee)
```
unless @colemickens doesn't want his affiliation mentioned.